composer.json

```diff
@@ -20,21 +20,22 @@
         }
     ],
     "require": {
-        "php":                          ">=5.5.9",
+        "php": ">=5.5.9",
 
-        "sylius/sylius":                "0.16.*@dev",
-        "sensio/generator-bundle":      "^2.3"
+        "sylius/sylius": "0.16.*@dev",
+        "sensio/generator-bundle": "^2.3",
+        "sensio/distribution-bundle": "^3.0"
     },
     "require-dev": {
-        "behat/behat":                  "^3.0",
-        "behat/symfony2-extension":     "^2.0",
-        "behat/mink-extension":         "^2.0",
+        "behat/behat": "^3.0",
+        "behat/symfony2-extension": "^2.0",
+        "behat/mink-extension": "^2.0",
         "behat/mink-browserkit-driver": "^1.2",
-        "behat/mink-selenium2-driver":  "^1.2",
-        "behat/mink":                   "^1.6",
-        "coduo/php-matcher":            "^1.0",
-        "phpspec/phpspec":              "^2.4",
-        "phpunit/phpunit":              "^4.1"
+        "behat/mink-selenium2-driver": "^1.2",
+        "behat/mink":  "^1.6",
+        "coduo/php-matcher": "^1.0",
+        "phpspec/phpspec": "^2.4",
+        "phpunit/phpunit": "^4.1"
```

Comment:
We are missing ApiTestCase and we should bump *php-matcher* to "^2.1@dev". Do we want this dependencies in sylius-standard?